designer/apps/pages/migrations/0006_auto_20190702_1505.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.21 on 2019-07-02 15:05
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pages', '0005_auto_20190701_1719'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='programpage',
+            name='uuid',
+            field=models.UUIDField(editable=False, unique=True),
```

Comment:
We're you able to save program pages with UUID being `editable=False`? I was going to remove the `editable=False` line in a followup PR.

Reply:
So right now we can't edit 'uuid' and program pages can only be made via the management command. I am open to changing this, but if you already have a PR ready with that change we can wait for that. Which ever

Reply:
I'll follow up on it. No need to add more work here.